browser-sync/browser-sync.d.ts

```diff
@@ -371,7 +373,7 @@ declare module "browser-sync" {
              * The stream method returns a transform stream and can act once or on many files.
              * @param opts Configuration for the stream method
              */
-            stream(opts?: { once: boolean }): NodeJS.ReadWriteStream;
+            stream(opts?: { once?: boolean, match?: string | RegExp | ((testString: string) => boolean) | (string | RegExp | ((testString: string) => boolean))[] }): NodeJS.ReadWriteStream;
```

Comment:
I think it would be appropriate to refactor this into its own interface, since the `match` type is very complex.  I would suggest `interface MatchOptions`.
Also, what is the difference between
`string | RegExp | ((testString: string) => boolean)` and `(string | RegExp | ((testString: string) => boolean)` such that you need a union between them?  The union types here are getting a bit verbose, which could confuse more than help when a user sees it in the autocompletion.  Perhaps a use case to illustrate?